homeassistant/components/habitica/strings.json

```diff
@@ -59,6 +59,14 @@
       }
     }
   },
+  "exceptions": {
+    "setup_rate_limit_exception": {
+      "message": "Currently rate limited, try again later"
+    },
+    "setup_authentication_exception": {
+      "message": "Authentication failed, verify you `User ID` and `API Token`"
+    }
```

Comment:
This should raise ConfigEntryAuthFailed and start reauth right?

Reply:
yeah, thats the plan but we don't have a reauth flow yet, so just raising ConfigEntryError for now

Reply:
But in that case we don't have to do this right now, we could do that ConfigEntryAuthFailed instead in a followup

Reply:
Hmm, thinking it through, the only possibility authentication fails at this point is, if someone requests by email that their API token gets revoked, otherwise it will already fail during initial configuration of the config entry.